FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SetUp/content.txt

```diff
@@ -1,7 +1,9 @@
 !|dbfit.TeradataTest|
 
+|Connect Using File|!-TestDbConnectionDbFitTeradata.properties-!|
```

Comment:
I like to keep the *graceful* fixture names (spaces). I've not tried `!|Connect Using File|`. I suspect it won't work. I'll try it.